modules/example/src/smithy4s/example/OrderNumber.scala

```diff
@@ -10,7 +10,7 @@ import smithy4s.schema.Schema.int
 object OrderNumber extends Newtype[Int] {
   val id: ShapeId = ShapeId("smithy4s.example", "OrderNumber")
   val hints: Hints = Hints(
-    smithy.api.Default(smithy4s.Document.fromDouble(0.0d)),
+    smithy.api.Box(),
```

Comment:
wut ?

Reply:
I believe this is from changing the smithy file in sampleSpecs over to use the smithy version 2 syntax directly. Prior it was a version 1 file being loaded which would have added these other traits.

Reply:
I thought the "box" trait had been removed :/

Reply:
Yeah.. this has really confused me before and I honestly don't remember what is going on here since it seems to conflict their documentation. But, here is an example using smithy-model version 1.27.2 (the latest)
![CleanShot 2023-02-06 at 09 37 29](https://user-images.githubusercontent.com/13733772/217030459-ab023e8b-6131-42bf-85f3-88a24a9f60a8.png)